woocommerce/woocommerce-ios
# Product Creation AI: Add new action sheet for product creation with AI

Closes: #10692 


## Description

This PR adds a new action sheet for an entry point to product creation AI flow. Changes include:
- New feature flag for product creation AI.
- UI for the new action sheet
- Add product: updated the coordinator to present the new action sheet based on the new feature flag. 

## Testing instructions

- Enable the feature flag `productCreationAI` and build the app.
- Log in to a new store without any product (excluding sample products for free trial stores).
- Tap the add new product task on store onboarding. Notice a new action sheet is displayed.
- Tap the manual option, an action sheet should be display for selecting product type.
- Switch to Products tab, select the "+" button to add a new product. The same action sheet should be displayed.

## Screenshots

<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/7298fbc3-1467-4cf9-8c50-3f671d5af592" width=320 />


Large font size:

https://github.com/woocommerce/woocommerce-ios/assets/5533851/3f5bf137-606a-4749-abfe-7400cb0ea6c1


---
